android/app/src/main/java/org/fossasia/openevent/core/auth/profile/EditProfileActivity.java

```diff
@@ -88,6 +93,17 @@ protected void onCreate(Bundle savedInstanceState) {
             Views.hideKeyboard(this, this.getCurrentFocus());
             saveChanges();
         });
+
+        ActivityCompat.requestPermissions(EditProfileActivity.this, READ_STORAGE, REQUEST_CODE);
+    }
+
+    @Override
+    public void onRequestPermissionsResult(int requestCode, String permissions[], int[] grantResults) {
+                if (grantResults.length > 0 && grantResults[0] == PackageManager.PERMISSION_GRANTED) {
+                    Toast.makeText(EditProfileActivity.this, "Access Successfull", Toast.LENGTH_SHORT).show();
```

Comment:
Why we are showing toast to the user, instead respective action should be taken for which the permission was sought. Also, the spelling of `successful` is wrong

Reply:
@iamareebjamal Actually, if the permission is granted the settings is automatically changed in the phone and thus no special action is required, so I thought a Toast message will thus notify the user

Reply:
@iamareebjamal Should I just remove the extra method involving the Toast messages? method `onRequestPermissionsResult`

Reply:
We should be taking the action when permission is granted. Not just silently passing it

Reply:
What type of action are you referring to?

Reply:
Like storage permissions are granted just after checking the dialog right? Is there a requirement of an other action?